multiuser/api/che-multiuser-api-resource/src/main/java/org/eclipse/che/multiuser/resource/api/usage/ResourceManager.java

```diff
@@ -125,4 +129,30 @@ public void checkResourcesAvailability(String accountId, List<? extends Resource
     // check resources availability
     resourceAggregator.deduct(availableResources, resources);
   }
+
+  /**
+   * Returns detailed information about resources which given account can use.
+   *
+   * @param accountId account id
+   * @return detailed information about resources which can be used by given account
+   * @throws NotFoundException when account with specified id was not found
+   * @throws ServerException when some exception occurs
+   */
+  public ResourcesDetails getByAccount(String accountId) throws NotFoundException, ServerException {
```

Comment:
Maybe it makes sense to rename it to `getResourcesDetails` or something like that. Because all method is this class return resources by account.